src/twisted/trial/_synctest.py

```diff
@@ -980,7 +980,7 @@ def getSkip(self) -> Tuple[bool, Optional[str]]:
         See L{TestCase} docstring for more details.
         """
         skipReason = util.acquireAttribute(self._parents, "skip", None)
-        doSkip = skipReason is not None
+        doSkip = skipReason not in [None, ""]
```

Comment:
`__contains__` runs `__eq__` and is quite different from `is not None`

Might be worth doing `doSkip = not skipReason` though anyway